tools/compile_tool/main.cpp

```diff
@@ -257,47 +264,107 @@ std::string getFileNameFromPath(const std::string& path,
     }
 }
 
+void printInputAndOutputs(const std::shared_ptr<const ov::Function>& network) {
+    std::cout << "Network inputs:" << std::endl;
+    for (auto&& input : network->inputs()) {
+        std::string layout;
+        if (auto parameter = dynamic_cast<const ov::op::v0::Parameter *>(input.get_node())) {
+            layout = parameter->get_layout().to_string();
+        }
```

Comment:
It looks a bit weird. Doesn't new API provide `input.get_layout()` method in the same way as `get_element_type`?

Reply:
Layout is not a port property. So here I have 2 ways, parse the port runtime info or ask parameter/result operations.